### DISCUSSION TITLE
[pimcore x] installer.yaml not working

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#9632] [Q&A / Support]
Hi,

I try to setup the pimcore install to use the config/installer.yaml file.

The config/installer.yaml looks like:

```
pimcore_install:
    parameters:
        database_credentials:
            user:                 %env(DB_USER)%
            password:             %env(DB_PASSWORD)%
            dbname:               %env(DB_NAME)%
            host:                 %env(DB_HOST)%
            port:                 %env(DB_PORT)%
```

But if I run the cpommand:

`docker-compose run --rm php vendor/bin/pimcore-install`

it always asks fo the db credentials....

Why?

**Comment by dvesh3** [comment]
try `docker-compose run --rm php vendor/bin/pimcore-install --skip-database-config`

**Reply by akno77** [reply]
I already tried this. Same result. It asks for the credentials...

**Reply by ShaileshT-dot** [reply]
can you try changing the name of file from installer.yaml to installer.yml? (app/config/installer.yml)

**Reply by dvesh3** [reply]
@ShaileshT-dot he is using correct file path for Pimcore X. It seems like a bug that `skip-database-config` option is ignored.

**Reply by ShaileshT-dot** [reply]
I referred to (https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Advanced_Installation_Topics.html) where it was mentioned as .yml file for Pimcore X.
Fine if the path is correct. Thanks.

**Reply by dvesh3** [reply]
sorry...you are right! this will change with 10.1 https://pimcore.com/docs/pimcore/10.1/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/index.html#page_10-1-0

@akno77 please try what @ShaileshT-dot suggested https://github.com/pimcore/pimcore/discussions/9632#discussioncomment-955486. thanks!
